scene/gui/graph_edit.cpp

```diff
@@ -1704,6 +1709,8 @@ void GraphEdit::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("_gui_input"), &GraphEdit::_gui_input);
 	ClassDB::bind_method(D_METHOD("_update_scroll_offset"), &GraphEdit::_update_scroll_offset);
+	ClassDB::bind_method(D_METHOD("is_in_input_hotzone", "graph_node", "child_index", "mouse_position"), &GraphEdit::is_in_input_hotzone);
+	ClassDB::bind_method(D_METHOD("is_in_output_hotzone", "graph_node", "child_index", "mouse_position"), &GraphEdit::is_in_output_hotzone);
```

Comment:
I don't think `child_index` is an appropriate name. Looking at where it comes from, `connection_index`, maybe?

Reply:
I thought `child index` was appropriate as I though it takes a child of a GraphNode for a slot to appear. For now, I've changed it to `slot_index` as I think it fits more.